### DISCUSSION TITLE
GitHub Support does not respond after ~3 months

### DISCUSSION THREAD

**Discussion by iamsharyn** [community/community#67427] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

For almost 3 months I haven't got any response for my support ticket. 

Looking at [previous discussions](https://github.com/orgs/community/discussions/43929), I understand the problem is not new (though of how common it is I am not sure). What's really annoying is that I got no other way to contact GitHub (e.g., in the past there was an email address which is now inactive), so I'm stuck in this situation. I even tried opening a separate ticket only for the first one to get attention.

Does anyone have an idea of what to do?

**Comment by kuroodo** [comment]
Migrate to GitLab

**Comment by chrisstef** [comment]
I am disappointed. It's been 1 month after my account was flagged as spam, for no obvious reason, and this has already costed me 2 job applications where I couldn't showcase my codebase and give access to a 3rd party app required for the application process. Github support hasn't reached out to me yet.
